concent_api/middleman/tests/test_middleman.py

```diff
@@ -8,34 +8,41 @@
 from assertpy import assert_that
 import mock
 import pytest
+from django.test import override_settings
+from golem_messages.message import Ping
 
+from common.testing_helpers import generate_ecc_key_pair
 from middleman.constants import DEFAULT_EXTERNAL_PORT
 from middleman.constants import DEFAULT_INTERNAL_PORT
 from middleman.constants import ERROR_ADDRESS_ALREADY_IN_USE
 from middleman.constants import LOCALHOST_IP
 from middleman.middleman_server import MiddleMan
+from middleman_protocol.message import GolemMessageFrame
```

Comment:
I think those imports should be where `golem_messages` imports are, before application specific import (common, middleman etc). Isn't link complaining about it ?

Reply:
Lint? No, no complains. But will do.

Reply:
Looks like pylint doesn't like general rules.
```
third party import "from django.conf import settings" should be placed before "from middleman_protocol.exceptions import ...
```
So I'll put `middleman_protocol` imports between django and application specific.

Reply:
Hmm weird, Django suggest this https://docs.djangoproject.com/ko/2.1/internals/contributing/writing-code/coding-style/ . But if lint is complaining than you should make it happy ;)

Reply:
That's probably because it's enforcing the general Python style:

[PEP8 > Imports](https://www.python.org/dev/peps/pep-0008/#imports)
> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.

But maybe `pylint-django` has this as an optional feature?